src/apps/mailing/static/templates/header.html

```diff
@@ -1,16 +1,3 @@
-<div style="margin-bottom: 30px">
-    <svg width="40" height="42" viewBox="0 0 40 42" fill="none"
-         xmlns="http://www.w3.org/2000/svg"
-         style="display: block; margin-left: auto; margin-right: auto">
-        <path d="M39.8882 42V0H30.2967L20.1071 19.6322L9.38815 0H0V42H8.74503V14.326L19.7963 35.6141L31.1465 14.326V42H39.8882Z"
-              fill="url(#paint0_linear_56222_71)"/>
-        <defs>
-            <linearGradient id="paint0_linear_56222_71" x1="19.9441" y1="0"
-                            x2="19.9441" y2="42"
-                            gradientUnits="userSpaceOnUse">
-                <stop stop-color="#28BAE9"/>
-                <stop offset="1" stop-color="#31E5C5"/>
-            </linearGradient>
-        </defs>
-    </svg>
-</div>
```

Comment:
The version in your screenshot seems to inherit the default font of the webmail interface – in your case, it's Times New Roman. Would it be possible to fix that by adding these lines to here?

```html
<style>
@import url('https://fonts.googleapis.com/css2?family=Atkinson+Hyperlegible:wght@400;700&display=swap');

body {
  font-family: "Atkinson Hyperlegible", sans-serif;
}
</style>
```

Reply:
Thanks @ChaconC! It's better, but in your example screenshot the whole text looks bold, when it seems like maybe it shouldn't? 🤔 The default text style in Figma looks [lighter than this](https://www.figma.com/file/hHPm6VvnfEyEcKvcVsg4UO/Transactional-emails?type=design&node-id=403-14&mode=design&t=SIW7ZoRFJ7xllXQU-4) by default, in contrast to the emboldened text.

Maybe I didn't get the Google Fonts URL quite right, or perhaps there just needs to be an explicit `font-weight: 400` as well?

Reply:
I think this may just be browser rendering vs Figma wonkiness. I added explicit `font-weight` 400 and 700 to make sure things are being loaded correctly:

![CleanShot 2024-02-26 at 12 40 54](https://github.com/ChildMindInstitute/mindlogger-backend-refactor/assets/2916156/297b4fcb-83f7-41f7-9ba8-7898f9f5ce25)